frb/figures/dm.py

```diff
@@ -266,6 +269,7 @@ def fig_cosmic(frbs, clrs=None, outfile=None, multi_model=False, no_curves=False
             Max redshift for the MR line
 
     Returns:
+        dict:
```

Comment:
And also ax optionally. Also, dict requires description.

Reply:
added